ohttp/src/rh/hpke.rs

```diff
@@ -54,15 +65,22 @@ impl Default for Config {
     }
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
Let's just go with this for now, but this might be better with boxing.  Having that be conditional on whether there are PQ keys is ugly, but maybe justifiable in this case.

Reply:
You'd say two kilobytes is too much for the stack?

Reply:
2kb isn't too much for the stack, but that isn't the real problem here.  clippy is complaining for a good reason: if you need just x25519, then 2kb allocation for that is straight up wasteful.

Reply:
Fair point.